File: face_comparsion/src/main/java/netposa/realtime/kafka/utils/RowMessage.scala
package netposa.realtime.kafka.utils

import kafka.common.TopicAndPartition

/**
 * 此class用于得到KAFKA的消息,并存储消息对应的TOPIC/PARTITION与当前消息的offset
 */
class RowMessage_bak(val tp:TopicAndPartition, val message:Array[Byte], val offset:Long) {
}

File: kafka_consumer_8/src/main/java/com/nukil/kafka/util/bean/RowMessage.java
package com.nukil.kafka.util.bean;

import kafka.common.TopicAndPartition;

public class RowMessage {
    private TopicAndPartition topicAndPartition;
    private byte[] message;
    private long offset;
    public RowMessage(TopicAndPartition topicAndPartition, byte[] message, long offset) {
        this.topicAndPartition = topicAndPartition;
        this.message = message;
        this.offset = offset;
    }
    public TopicAndPartition topicAndPartition() {
        return this.topicAndPartition;
    }
    public byte[] message() {
        return this.message;
    }
    public long offset() {
        return this.offset;
    }
}